CONTRIBUTING.md

```diff
@@ -68,10 +68,18 @@ poetry run flake8 ./xrpl
 
 ### Running Tests
 
+#### Individual Tests
+
+```bash
+# Works for single or multiple unit/integration tests
+# Ex: poe test tests/unit/clients/test_json_rpc_client.py tests/integration/transactions/test_account_delete.py
+poe test FILE_PATHS
+```
+
 #### Unit Tests
 
 ```bash
-poetry run python3 -m unittest discover tests/unit
+poe test_unit
```

Comment:
```suggestion
poetry run poe test_unit
```
Poetry is a Python package manager - it handles the package installs itself. Contributors don't need to install it themselves.

ditto with below